src/ratingsAndReviews/components/reviewList.jsx

```diff
@@ -77,10 +90,11 @@ class ReviewList extends React.Component {
               sortNew={this.sortByNewest.bind(this)}
             />
           </div>
-          <ReviewTile product_data1={this.props.productInfo} />
+          <ReviewTile product_data1={this.props.productInfo} index={this.state.index} />
           <div className="rar_reviewButtons">
 
-            <button className="button" type="submit" id="moreReviews">MORE REVIEWS</button>
+            <button className="button" type="submit" id="moreReviews" onClick={this.updateIndex.bind(this)}>MORE REVIEWS</button>
```

Comment:
Can you move the actual bindings of these methods to the constructor?

Reply:
Changes have been made as suggested. Thank you for the feedback!